# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1252101466 14400
#      Fri Sep 04 17:57:46 2009 -0400
# Node ID 7c628fb04fd077a968034b59444fb37d95c38032
# Parent  2093499ec9f4b3e0c2c52ced20f5ea2cc7b15baa
Makeconf.in: don't sort sparse and fftw cpp/ld flags

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-09-04  John W. Eaton  <jwe@octave.org>
+
+	* Makeconf.in (SPARSE_CPPFLAGS, SPARSE_LDFLAGS, FFTW_CPPFLAGS,
+	FFTW_LDFLAGS): Don't sort lists.
+
 2009-09-02  Marco Atzeri  <marco_atzeri@yahoo.it>
 
 	* Makeconf.in: Order SPARSE_LIBS so linking succeeds on Cygwin.
 
 2009-09-04  Marco Atzeri  <marco_atzeri@yahoo.it>
 
 	*  README.Cygwin: Update version info.
 
diff --git a/Makeconf.in b/Makeconf.in
--- a/Makeconf.in
+++ b/Makeconf.in
@@ -303,39 +303,35 @@ ARPACK_CPPFLAGS = @ARPACK_CPPFLAGS@
 ARPACK_LDFLAGS = @ARPACK_LDFLAGS@
 ARPACK_LIBS = @ARPACK_LIBS@
 
 DL_LIBS = @DL_LIBS@
 LIBS = @LIBS@
 
 ALL_CPPFLAGS = $(CPPFLAGS) $(HDF5_CPPFLAGS) $(Z_CPPFLAGS)
 
-## Use sort to remove duplicates.
 SPARSE_CPPFLAGS = \
-  $(sort $(AMD_CPPFLAGS) $(CAMD_CPPFLAGS) $(COLAMD_CPPFLAGS) \
-    $(CCOLAMD_CPPFLAGS) $(CHOLMOD_CPPFLAGS) $(CXSPARSE_CPPFLAGS) \
-    $(UMFPACK_CPPFLAGS))
+  $(CHOLMOD_CPPFLAGS) $(UMFPACK_CPPFLAGS) \
+  $(AMD_CPPFLAGS) $(CAMD_CPPFLAGS) $(COLAMD_CPPFLAGS) \
+  $(CCOLAMD_CPPFLAGS) $(CXSPARSE_CPPFLAGS)
 
-## Use sort to remove duplicates.
 SPARSE_LDFLAGS = \
-  $(sort $(AMD_LDFLAGS) $(CAMD_LDFLAGS) $(COLAMD_LDFLAGS) \
-    $(CCOLAMD_LDFLAGS) $(CHOLMOD_LDFLAGS) $(CXSPARSE_LDFLAGS) \
-    $(UMFPACK_LDFLAGS))
+  $(CHOLMOD_LDFLAGS) $(UMFPACK_LDFLAGS) \
+  $(AMD_LDFLAGS) $(CAMD_LDFLAGS) $(COLAMD_LDFLAGS) \
+  $(CCOLAMD_LDFLAGS) $(CXSPARSE_LDFLAGS)
 
 ## Order matters, at least on some systems (Cygwin, for example).
 SPARSE_LIBS = \
     $(CHOLMOD_LIBS) $(UMFPACK_LIBS) \
     $(AMD_LIBS) $(CAMD_LIBS) $(COLAMD_LIBS) \
     $(CCOLAMD_LIBS) $(CXSPARSE_LIBS)
 
-## Use sort to remove duplicates.
-FFTW_CPPFLAGS = $(sort $(FFTW3_CPPFLAGS) $(FFTW3F_CPPFLAGS))
+FFTW_CPPFLAGS = $(FFTW3_CPPFLAGS) $(FFTW3F_CPPFLAGS)
 
-## Use sort to remove duplicates.
-FFTW_LDFLAGS = $(sort $(FFTW3_LDFLAGS) $(FFTW3F_LDFLAGS))
+FFTW_LDFLAGS = $(FFTW3_LDFLAGS) $(FFTW3F_LDFLAGS)
 
 FFTW_LIBS = $(FFTW3_LIBS) $(FFTW3F_LIBS)
 
 USE_64_BIT_IDX_T = @USE_64_BIT_IDX_T@
 
 TEXINFO_COLAMD = @TEXINFO_COLAMD@
 TEXINFO_CHOLMOD = @TEXINFO_CHOLMOD@
 TEXINFO_UMFPACK = @TEXINFO_UMFPACK@
